AlphaWallet/Settings/Types/Constants.swift

```diff
@@ -10,6 +10,19 @@ public struct Constants {
         Array(arrayLiteral: 0x58, 0x44, 0x41, 0x49, 0x44, 0x52, 0x4F, 0x50)
     ).hex()
 
+    // Magic link networks
+    public static let mainnetMagicLinkPrefix = "https://app.awallet.io/"
```

Comment:
Shall we add a https://eth.awallet.io/ and make it compatible with "https://app.awallet.io/" but no longer generate `app.awallet.io` links at some point (after all mobile apps are updated)? That could free up the `app.awallet.io` sub-domain later.